.github/issue_template.md

```diff
@@ -1,7 +1,7 @@
-## Issue Overview
-
-Provide a brief summary of the issue, including the motivation for addressing it, the problems it presents, and any features required to resolve it.
-
-## Proposed Solutions or Ideas
-
-Outline your proposed solution or share any ideas you have for discussing or resolving the issue.
+## Issue Overview

```

Comment:
Please revert all the changes that are not in scope of the issue, thank you!

Reply:
@metalboyrick @Nadai2010 Sorry, I'm not sure what extra files I might have touched. Are you referring to the theme changer that I adjusted? Could you please clarify which specific changes are out of scope for this issue? I'd be happy to revert any unintended modifications once I understand exactly what needs to be changed back.

Reply:
i think i have fixed it, please tell me if there's an issue

Reply:
Hello @omsant02 when making the formatting correction, many unnecessary changes were pushed. That format should be fine just by cloning. 

Only the changes to the necessary files should be pushed (as was done the first commit), but with `yarn format` so that the code syntax is correctly formatted to those new changes. Do you understand?